### PR TITLE
Add support for org claim

### PR DESCRIPTION
**Pull request by pkoenig10**
Add support for the `org` claim we are adding to our JWTs. This claim is also a Base64-encoded UUID.

For more context, see https://g.p.b/foundry/mp/pull/16056.

## Possible downsides?

This may break users of this library whose JWTs have an `org` claim that is not a Base64-encoded UUID. But we've never claimed that this library should be used for arbitrary JWTs - it's only intended to be used for Palantir JWTs.

### CODE REVIEW

**Comment by pkoenig10** [comment]
Do not merge until we've made the internal change to begin producing this claim.

File: auth-tokens/src/test/java/com/palantir/tokens/auth/UnverifiedJsonWebTokenTests.java
Comment (pkoenig10) [inline auth-tokens/src/test/java/com/palantir/tokens/auth/UnverifiedJsonWebTokenTests.java:29]: These tests are currently designed based on our internal token types (session vs API). This is an implementation details that consumers should not be concerned with. I've refactored these tests to better reflect the guarantees and assumptions of this library itself.

**Review by carterkozak — APPROVED** [review]
Thanks, I appreciate the updated documentation as well

**Comment by carterkozak** [comment]
Feel free to merge this when you're ready

**Pull request merged**
